### PR TITLE
New gh-pages build

### PR DESCRIPTION
**Pull request by eduardojmatos**
**CHANGELOG** :memo:

**New binary build documentjs!**
Now we have a gh-pages available to our designers 😎🤓;

The URL is https://getninjas.github.io/gaiden

### CODE REVIEW

**Comment by eduardojmatos** [comment]
[(2) Criação de documentação online do Gaiden v1](https://trello.com/c/SjuTIZju/387-2-criacao-de-documentacao-online-do-gaiden-v1)

File: bin/docs/prod/docs.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env node
+
+const DocsGenerator = require('./docs-generator');
```
Comment (iondrimba) [inline bin/docs/prod/docs.js]: 💯

File: bin/docs/dev/docs.js [hunk 2]
```diff
@@ -16,6 +16,6 @@ const options = {
     --recursive `
 }
 
-shelljs.exec(`node-sass ${sourceDocsDir} ${options.sass} ${minifiedDocsFile}`);
-shelljs.exec(`documentjs`);
-shelljs.cp('./build/stylesheets/gaiden.css', `${docsDir}/`)
+sh.exec(`node-sass ${sourceDocsDir} ${options.sass} ${minifiedDocsFile}`);
+sh.exec(`documentjs`);
+sh.cp('./build/stylesheets/gaiden.css', `${docsDir}/`)
```
Comment (iondrimba) [inline bin/docs/dev/docs.js]: ; 💅

**Comment by iondrimba** [comment]
@eduardojmatos Could you add link to the live docs on the README?

**Pull request merged**